src/shared/components/VideoPlayer/CustomTimeline.style.ts

```diff
@@ -41,35 +46,38 @@ export const ProgressControl = styled.div<ProgressControlProps>`
     opacity: 1;
   }
   :hover ${() => MouseDisplayTooltip} {
-    transform: translateY(-0.5em) !important;
+    transform: translateY(-0.5em);
     opacity: 1;
   }
   :hover ~ ${CustomControls} {
     opacity: 0;
-    transform: translateY(0.5em) !important;
+    transform: translateY(0.5em);
   }
 
   ${() => SeekBar} {
-    ${({ isScrubbing }) => isScrubbing && `height: 0.5em`}
+    ${({ isScrubbing }) => isScrubbing && `height: 0.5em;`}
   }
 
   ${() => MouseDisplayWrapper}, ${() => PlayProgressThumb} {
-    ${({ isScrubbing }) => isScrubbing && `opacity: 1`}
+    ${({ isScrubbing }) => isScrubbing && `opacity: 1;`}
   }
   ${() => MouseDisplayTooltip} {
-    ${({ isScrubbing }) => isScrubbing && `transform: translateY(-0.5em) !important`}
+    ${({ isScrubbing }) => isScrubbing && `transform: translateY(-0.5em);`}
   }
   ~ ${CustomControls} {
-    ${({ isScrubbing }) => isScrubbing && `opacity: 0; transform: translateY(0.5em) !important`};
+    ${({ isScrubbing }) => isScrubbing && `opacity: 0; transform: translateY(0.5em);`}
   }
```

Comment:
Maybe it would be more clear to define those scrubbing styles in a separate block? Like so:

```jsx
const scrubbingStyles = css`
  ${() => SeekBar} {
    height: 0.5em;
  }
  ${() => MouseDisplayWrapper}, ${() => PlayProgressThumb} {
    opacity: 1;
  }
  // ...
`

export const ProgressControl = styled.div<ProgressControlProps>`
  ${({ isScrubbing }) => isScrubbing && scrubbingStyles};
`
```

What do you think?

Reply:
I don't think it's possible with emotion `css`. Typescript is throwing me errors, and if I ignored these errors I would get:
![image](https://user-images.githubusercontent.com/51168865/127863852-620016d2-4eb7-41a9-ae5a-de12907e6410.png)

To make this work I would have to pass component as an argument to `scrubbingStyles`, something like this: 
```tsx
const scrubbingStyles = (seekBar: typeof SeekBar, /* other components here */ isFullScreen?: boolean) => css`
  height: 100vh;
  bottom: ${isFullScreen ? 0 : '-200px'};
  padding-bottom: ${isFullScreen ? `1.5em 1.5em` : '200px'};
  ${seekBar} {
    height: 0.5em;
  }

  /* other components here */
`
```